src/DataMatrix.java

```diff
@@ -115,25 +115,7 @@ public void displayImageToConsole() {
    private void cleanImage() {
       int startRow = 0;
       int startColumn = 0;
-      boolean spine = false;
-
-      while (!spine) {
-         for (int i = BarcodeImage.MAX_HEIGHT; i > 0; i--) {
```

Comment:
@harsandeep1 when I said to remove the while loop, i meant to remove line 120 and just have the for loops. You still need the for loops but you don't need the while loop on the outside. So sorry for the confusion again

Reply:
if you can put lines 121-129 back that would be sufficient